explosion.py

```diff
@@ -0,0 +1,29 @@
+from constants import *
+
+class Explosion(pygame.sprite.Sprite):
+
+    def __init__(self, center_x, center_y, scale: tuple = (100, 100)) -> None:
```

Comment:
not sure how this works but I'm excited to learn!

Reply:
Are you unsure of the constructor?  I'll see if I can clean up the code to be more readable.